### DISCUSSION TITLE
Syntax highlighting for no-suffix file

### DISCUSSION THREAD

**Discussion by Yuhyeong** [zed-industries/zed#16146] [Feature Requests]
```json
  "file_types": {
    "Shell Script": ["*."]
  }
```
How to set file with no suffix as shell, so that i could be highlightened. I tried settings above, doesnt work. And when I set *, all the file include specific language type are covered by this settings which caused wrong highlighten method.

**Comment by steezeburger** [comment]
The syntax highlighting is correct for my bash files that have no suffix, but the ui does not render the correct icon for bash files. It would be nice to even be able to set filetypes for individual files.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
